optimum/onnxruntime/modeling_ort.py

```diff
@@ -1278,6 +1286,7 @@ class ORTModelForImageClassification(ORTModel):
     """
 
     auto_model_class = AutoModelForImageClassification
+    _library_name: Optional[str] = "transformers"
```

Comment:
also compatible for timm models so should be removed here 
```suggestion
```

Reply:
Nice catch!